test/src/receiver/main_receiver.cpp

```diff
@@ -20,10 +20,14 @@
 #include <cadmium/engine/pdevs_dynamic_runner.hpp>
 #include <cadmium/logger/tuple_to_ostream.hpp>
 #include <cadmium/logger/common_loggers.hpp>
-#include "../../../lib/vendor/NDTime.hpp"
+#include "../../../lib/DESTimes/include/NDTime.hpp"
 #include "../../../lib/vendor/iestream.hpp"
 #include "../../../include/data_structures/message.hpp"
 #include "../../../include/atomics/receiver_cadmium.hpp"
+#include "../../../include/output_modification/convert_output.hpp"
+
+#define SIMULATOR_OUTPUT_PATH "./test/data/receiver_test_output.txt"
+#define MODIFIED_OUTPUT_FILE "./test/data/modified/converted_receiver_output.txt"
```

Comment:
name of #define is not self-explanatory. This is receiver component and #define statement must be consistent with it.

eg RECEIVER_MODIFIED_OUTPUT_FILE_PATH